src/main/java/sg/ncl/MainController.java

```diff
@@ -855,12 +855,12 @@ private void registerUserToDeter(JSONObject mainObject) throws
 
                 ExceptionState exceptionState = ExceptionState.parseExceptionState(error.getError());
 
-                final String errorPrefix = "Error: ";
+                final String ERROR_PREFIX = "Error: ";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "ERROR_PREFIX" which hides the field declared at line 80. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)